brambling/views/core.py

```diff
@@ -47,7 +47,8 @@ def get_context_data(self):
             re_dict = dict((e.pk, e) for e in registered_events)
             orders = Order.objects.filter(
                 person=user,
-                bought_items__status=BoughtItem.RESERVED,
+                bought_items__status__in=(BoughtItem.BOUGHT,
+                                          BoughtItem.RESERVED),
```

Comment:
`BOUGHT` is something that has been paid for; `RESERVED` is in the cart (and potentially expiring); `UNPAID` is a special status that counts as in the cart but doesn't get expired.